src/core/Fieldset/Fieldset.test.tsx

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { Fieldset } from './Fieldset';
+
+it('should render in its most basic state', () => {
+  // TODO: Make sure all required props are passed in here
+  const { container } = render(<Fieldset />);
+  expect(container.querySelector('div')).toBeTruthy();
+});
+
+// TODO: Write tests here!
+
+it('should be improved', () => {
+  expect(false).toBe(true);
+});
```

Comment:
Test for:
1. basic state (check classname and element name)
2. disabled state (also check that it sets disabled on children)
3. custom props passed by user (e.g. id, style)